packages/headless/src/controllers/relevance-inspector/headless-relevance-inspector.ts

```diff
@@ -59,6 +65,14 @@ export interface RelevanceInspectorInitialState {
  * The `RelevanceInspector` controller is in charge of allowing displaying various debug information.
  */
 export interface RelevanceInspector extends Controller {
+  /**
+   * Fetch the description of all fields available from the index.
+   */
+  fetchFieldsDescription(): void;
+  /**
+   * Fetch all fields available from the index on each individual results.
+   */
+  debugFields(fetchAll: boolean): void;
```

Comment:
I think i'd vote for the enable/disable methods

Reply:
Yea, maybe `enableFetchAllFields` and `disableFetchAllFields`?

Having them part of the relevance inspector controller is ok, though I think a `FieldInspector` controller makes a lot of sense? When someone instantiates it, it could also auto fetch the field descriptions, expose properties in state that group by facet type, sort etc.